source/_integrations/totalconnect.markdown

```diff
@@ -21,19 +21,100 @@ ha_integration_type: integration
 
 The `totalconnect` integration provides connectivity with Resideo Total Connect 2.0-enabled alarm systems.
 
+The integration allows automations based on the state of the alarm system. For example: when the alarm is armed, turn on the outside light.
+
+The integration can also arm or disarm the alarm panel.  For example: when I arrive home, disarm the alarm panel
+
+## Supported devices
+
+This integration supports alarm panels with "communicator modules" that are compatible with the Total Connect 2.0 service.  An active account is required via a third party alarm monitoring company. Panels may be branded as Resideo, Honeywell, Ademco or other third party alarm companies.
+
+Device models known to work include:
+
+- ProSeries (ProA7, ProA7Plus)
+- Lyric (AIO, Gateway)
+- Lynx Touch (L5100, L5200, L5210, L7000)
+- Lynx Plus (L3000)
+- VISTA (15P, 20P, 21iP)
+
+## Unsupported devices
+
+The following devices are not supported:
+
+- Older Lynx models (not Touch or Plus)
+- Other panels without a Total Connect compatible communicator module
+
 ## Prerequisites
 
 Log in to the [Total Connect website](https://totalconnect2.com) and create a "standard" Total Connect user account specifically for use with Home Assistant. It should not have full administrative privileges.
 
 Give the user access to your Location, along with a user code, usually a 4 digit number.
 
+{% details "Notes for Home Assistant Core Installations" %}
+
+If you have issues running this integration, you may require `libxml2-dev` and `libxmlsec1-dev` packages. To install these on Raspbian, run the command:
```

Comment:
Any specific reason we mention raspbian? Is this installed on other linux distros by default?

Note that apparently Raspberry Pi renamed Raspbian to Raspberry Pi OS https://www.raspberrypi.com/software/